<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTile">
     <property name="styleSheet">
      <string notr="true">font: 75 26pt &quot;微软雅黑&quot;;
background-color: rgb(73, 169, 242);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>悬浮滚动条</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="W_CustomListWidget" name="listWidget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="W_ListWidget" name="listWidget_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SuspendedScrollBar_Area" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>750</width>
            <height>414</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>541</width>
           <height>384</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>541</width>
           <height>384</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/zf.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font:14px &quot;微软雅黑&quot;;
	color:#49a9f2;
}
</string>
         </property>
         <property name="text">
          <string>如果有问题或者建议
欢迎跟我联系
QQ：392139105


如果对你有帮助
可以通过左边二维码打赏我</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(73, 169, 242);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnList">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font:14px &quot;微软雅黑&quot;;
	color:#feffff;
	background:transparent;
	border-radius:4px;
	border:1px solid #ffffff;
}
QPushButton:hover{
	font:14px &quot;微软雅黑&quot;;
	color:#49a9f2;
	background:#ffffff;
	border-radius:4px;
}</string>
        </property>
        <property name="text">
         <string>列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImage">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font:14px &quot;微软雅黑&quot;;
	color:#feffff;
	background:transparent;
	border-radius:4px;
	border:1px solid #ffffff;
}
QPushButton:hover{
	font:14px &quot;微软雅黑&quot;;
	color:#49a9f2;
	background:#ffffff;
	border-radius:4px;
}</string>
        </property>
        <property name="text">
         <string>图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font:14px &quot;微软雅黑&quot;;
	color:#feffff;
	background:transparent;
	border-radius:4px;
	border:1px solid #ffffff;
}
QPushButton:hover{
	font:14px &quot;微软雅黑&quot;;
	color:#49a9f2;
	background:#ffffff;
	border-radius:4px;
}</string>
        </property>
        <property name="text">
         <string>联系我</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>W_CustomListWidget</class>
   <extends>QListWidget</extends>
   <header>w_customlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>SuspendedScrollBar_Area</class>
   <extends>QScrollArea</extends>
   <header>suspendedscrollbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>W_ListWidget</class>
   <extends>QListWidget</extends>
   <header>w_customlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
